StratisCore.UI/src/app/setup/recover/recover.component.ts

```diff
@@ -26,9 +26,11 @@ export class RecoverComponent implements OnInit {
   public minDate = new Date("2009-08-09");
   public maxDate = new Date();
   public bsConfig: Partial<BsDatepickerConfig>;
+  public sidechainEnabled: Boolean;
```

Comment:
I think the best practive is not to use Boolean (the one with capital B), because
```
var falseObj = new Boolean(false);
if (falseObj) {
    console.log('our intent may not be to get here');
}
```